app/models/curator.rb

```diff
@@ -46,6 +47,14 @@ def is_user?(user)
     self.id == user.id
   end
 
+  def has_pending_sightings?
+    self.collection_observations.where(pending: true).order(updated_at: :desc).any?
```

Comment:
You can probably take out the updated_at? in this line, since this method is returning true or false